js/player.js

```diff
@@ -163,10 +164,8 @@ function($, playlist, ytIframe){
         savePlaying: savePlaying,
         setPlaying: setPlaying,
         getCurrent: function() {
+            playing=parseInt(localStorage.getItem('playing'));
             return playlist.get(playing);
-        },
-        getPlaying: function(){
-            return playing;            
```

Comment:
Don't undo the change introduced in #32. Because it is merged, you should rebase to master.